mmengine/config/config.py

```diff
@@ -1605,6 +1609,36 @@ def merge_from_dict(self,
             Config._merge_a_into_b(
                 option_cfg_dict, cfg_dict, allow_list_keys=allow_list_keys))
 
+    @staticmethod
+    def diff(cfg1: Union[str, 'Config'], cfg2: Union[str, 'Config']) -> str:
+        if isinstance(cfg1, str):
+            cfg1 = Config.fromfile(cfg1)
+
+        if isinstance(cfg2, str):
+            cfg2 = Config.fromfile(cfg2)
+
+        res = difflib.unified_diff(
+            cfg1.pretty_text.split('\n'), cfg2.pretty_text.split('\n'))
+
+        # Convert into rich format for better visualization
+        console = Console()
+        text = Text()
+        for line in res:
+            if line.startswith('+'):
+                color = 'bright_green'
+            elif line.startswith('-'):
+                color = 'bright_red'
+            else:
+                color = 'bright_white'
+            line = Text(line + '\n')
+            line.stylize(color)
+            text.append(line)
```

Comment:
```suggestion
            _text = Text(line + '\n')
            _text.stylize(color)
            text.append(_text)
```

Reply:
The above modifications may pass the lint.

Reply:
Thanks. the first trick worked.